app/grandchallenge/profiles/models.py

```diff
@@ -104,7 +104,7 @@ def get_absolute_url(self):
 
     def get_mugshot_url(self):
         if self.mugshot:
-            return self.mugshot.x02.url
+            return self.mugshot.x20.url
```

Comment:
Is this intentional or a typo? Everywhere else we seem to be referencing `x20`. `x02` returned a blurry image.

Reply:
Clear. I'll change the forum implementation then.